README.md

```diff
@@ -29,23 +30,54 @@ Installs Splunk or Universal Forwarder.
-| package | `:splunk` or `:universal_forwarder` | **Yes**  |         | Specifies the Splunk package to uninstall. You must specify the package, or name the resource for the package; for example, `package :splunk` or `splunk_install 'universal_forwarder' do ... end`. |
+| Name    |               Type(s)               | Required | Default | Description                                                                                                                                                                                        |
+|:--------|:-----------------------------------:|:--------:|:--------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| package | `:splunk` or `:universal_forwarder` | **Yes**  |         | Specifies the Splunk package to uninstall. You must specify the package, or name the resource for the package; for example, `package :splunk` or `splunk_install 'universal_forwarder' do ... end` |
 
 
+### splunk_service
+Manages an installation of Splunk
+
+**The following actions share the same properties:**
+
+##### Action *:start*
+Starts the Splunk daemon if not already running.
+If the ulimit is changed, invokes a restart of the daemon at the end of the run.
+
+##### Action *:restart*
+Restarts the Splunk daemon, or starts it if not already running.
+
+Properties:
+
+| Name    |               Type(s)               | Required | Default                                                         | Description                                                                                                                                                                                      |
+|:--------|:-----------------------------------:|:--------:|:----------------------------------------------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| package | `:splunk` or `:universal_forwarder` | **Yes**  |                                                                 | Specifies the Splunk package to install. You must specify the package, or name the resource for the package; for example, `package :splunk` or `splunk_install 'universal_forwarder' do ... end` |
+| user    |               String                |    No    | Owner of the specified Splunk installation, or the current user | User to run Splunk as. This is the user that will be used to run the Splunk service.                                                                                                             |
```

Comment:
The knee-jerk reaction here was to use the user used by the install resource (probably communicated by the run_state)... Given how Splunk will write files to it's own directory, it can be problematic to run splunk as a different user than an installed user... For example if you do the first time run step as the root user, instead of the Splunk user, then start splunk as the splunk user, certain created files would not be accessible by Splunk and you wind up in a problematic state.